swagger-future.yml

```diff
@@ -978,6 +980,7 @@ components:
       properties:
         caseId:
           type: string
+          format: uuid
```

Comment:
not your problem, but since we are lining up future and current swagger, extra browny points for removing the extra white space at the end of line 969, so that the the two files are even closer to each other!

Reply:
Fair point.
Done